### PR TITLE
Updates amount format

### PR DESCRIPTION
**Pull request by joaomilho**

### CODE REVIEW

File: integration.md [hunk 1]
```diff
@@ -26,19 +26,19 @@ While the specifics vary slightly depending on your use case, there are three ma
 1. Use a script tag pointing to Klarna's SDK. There are two versions of the SDK, one that works with our playground environment and one that works with our production environment. Other than the environment they are geared towards, both are functionally equivalent.
 
    For the playground version, use:
-   
+
```
Comment (joaomilho) [inline integration.md:29]: Sorry, but I can't live with empty spaces.

File: integration.md [hunk 2]
```diff
@@ -218,5 +218,5 @@ Attribute | Description
 data-api-key | This is where you must supply your API key for the form to function properly.
 data-flow | The type of on-demand form to display. The supported values are `purchase` and `recurring-purchase`
 data-locale | The locale in which the form should be presented.
-data-amount | This attribute is valid only for purchase forms and denotes the purchase's total price.
+data-amount | This attribute is valid only for purchase forms and denotes the purchase's total price. The value should be multiplied by 100, so if your price is "9,90" it should be sent as "990".
```
Comment (yuval-netanel) [inline integration.md:221]: You can say that it should be in the ISO 4217 (https://en.wikipedia.org/wiki/ISO_4217) format and in this example you should multiply by 100

**Comment by yuval-netanel** [comment]
:+1:

File: integration.md (hunk 2 above)
Comment (joaomilho) [inline integration.md:221]: :+1: will do

Comment (joaomilho) [inline integration.md:221]: It's already in the line below about currency actually.

**Pull request merged**